amd_openvx_extensions/amd_media/decoder.cpp

```diff
@@ -741,6 +754,11 @@ void CLoomIoMediaDecoder::DecodeLoop(int mediaIndex)
                 return;
             }
             int status = avcodec_receive_frame(videoCodecContext[mediaIndex], frame);
+#if ENABLE_PERF_MEASURE           
```

Comment:
@paveltc this PR is the same as PR#962. please take a look to my comments for PR#962 and make the requested changes. Thanks.

Reply:
@asalmanp Looking into it. Somehow the older version of the file got added.